src/avkit.cs

```diff
@@ -1099,4 +1100,36 @@ interface AVPlaybackSpeed {
 		[Export ("localizedNumericName")]
 		string LocalizedNumericName { get; }
 	}
+
+	[iOS (17, 2), NoMac, NoMacCatalyst, NoTV, NoWatch]
+	[BaseType(typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVCaptureEvent
+	{
+		[Export("phase")]
+		AVCaptureEventPhase Phase { get; }
+	}
+
+	[iOS (17, 2), NoMac, NoMacCatalyst, NoTV, NoWatch]
+	[Native]
+	public enum AVCaptureEventPhase : ulong {
```

Comment:
I noticed there's a file at `src/AVKit/Enums.cs`, but there are just as many enums in `src/avkit.cs`.  I wasn't sure which file I should use, so I just put the enum here. I can move it to the enum file if that's preferred though.

Reply:
The split is just for historical reasons, in `avkit.cs` is fine.